### PR TITLE
Update MLE Lecture with More Details

### PR DESCRIPTION
**Pull request by HumphreyYang**
Hi @jstac,

I added more details about the likelihood and log likelihood functions of Poisson regression. I think MLE can be a good example for a lecture on symbolic algebra since it has a routine that can be tedious to calculate by hand sometimes. It is also conceptually easy so that people can focus on the implementation. 

Please kindly review this PR and let me know if there is anything I can further improve.

Many thanks.

### CODE REVIEW

File: lectures/mle.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ kernelspec:
   name: python3
 ---
 
++++ {"user_expressions": []}
```
Comment (jstac) [inline lectures/mle.md:14]: Please delete these.

**Comment by jstac** [comment]
Thanks @HumphreyYang , I like the way you added more explanations.  They will be helpful to the reader.

Please see my minor comments above.

**Comment by HumphreyYang** [comment]
Hi @jstac,

> Please see my minor comments above.

They are all removed now.

Many thanks for your review.

**Comment by jstac** [comment]
Nice work @HumphreyYang , many thanks.

**Pull request merged**